bigflow/scaffold/templates/new-project/resources/requirements.txt

```diff
@@ -0,0 +1,81 @@
+# *** AUTO GENERATED: DON'T EDIT ***
+# $source-hash: sha256:536b5d9967c9983def6b9ee11127963a5a209d1713d6c4a90114c0789fb0c373
+# $source-file: resources/requirements.in
+#
+# Run 'bigflow pip-compile resources/requirements.in' to update this file
```

Comment:
What about the `run` method? Maybe we should also check the requirements freshness during a run and automatically upgrade requirements (I got that request from our client)?

Reply:
Added simple check (with hardcoded 'resources/requirements.txt'
However it doesn't verify that installed libs are in sync with 'requirements.txt'
We can do this with 'pip-sync' command (part of used 'pip-tools'), but probably this is out of scope of this PR (and current in-dev version 1.1).